db/migrate/20140716131107_create_category_translation_tables.rb

```diff
@@ -136,16 +136,16 @@ def down
     new_headings = []
     PublicBodyHeading.all.each do |heading|
       heading.locale = heading.translation.locale.to_s
-      FastGettext.default_available_locales.each do |locale|
-        if locale.to_s != heading.locale
+      AlaveteliLocalization.available_locales.each do |locale|
+        if locale != heading.locale
```

Comment:
It's a `String`, it's not translated, it's being read from the database column we've just [re]created